src/maat/web/templates/index.html

```diff
@@ -63,6 +63,18 @@
 <body>
 <header>
     <h1>Ma'at</h1>
+
+    <ul id="slices" style="list-style-type: none; display: flex; gap: 1em; justify-content: center;">
+    {% for slice in slices %}
+        <li>
+            <a
+                href="{{ slice.href }}"
+                class="{{ "button"|clsx(current=slice.is_current) }}">
+                {{ slice.title }}
```

Comment:
```suggestion
                {{- slice.title -}}
```

HTML is sometimes whitespace-sensitive. I don't want to go too deep into the details, but in this scenario (a _prose-style_ element) this whitespace would end up as real space characters in rendered view.

`{{- -}}` tells jinja to remove whitespace before/after this expression, so post-jinja this will be

```html
<a href="..."
   class="...">slice title</a>
```